src/components/connect-wallet/ConnectWallet.scss

```diff
@@ -36,6 +36,7 @@ div.connectWallet {
 
       &:hover {
         cursor: pointer;
+        background-color: var(--chakra-colors-purple-200);
```

Comment:
We would want to move way from chackra, consider changing the color name to something more inline with axial.

Reply:
Okay. Here's something more inline with axial. I changed the button + hover state to match axial's primary button + hover state.

Axial's buttons:

Default
<img width="210" alt="image" src="https://user-images.githubusercontent.com/23615754/142298561-ffe4afe5-53f8-48da-8ff4-accd1abf558e.png">

Hover State:
<img width="211" alt="image" src="https://user-images.githubusercontent.com/23615754/142298584-ec1a28de-2852-4069-9e31-0a2b1ffd9a5c.png">

Connect wallet buttons:

Default:
<img width="485" alt="image" src="https://user-images.githubusercontent.com/23615754/142298613-a0690c91-bc6b-4cb3-a3b7-a141edc60efc.png">

Hover State:
<img width="486" alt="image" src="https://user-images.githubusercontent.com/23615754/142298655-bd095d38-2172-44d1-9fd6-d7416dc9a713.png">

We can do something like that to really be aligned with Axial's theme - or, if that's too much, I can literally just change the color name from `--chakra-colors-purple-200` to whatever its hex value is and call it a day.

Reply:
Hmm, although now that I'm looking at it, I don't know if I like what the new color does to the WalletConnect icon (that contrast isn't that great). Maybe I should just stick with the `--chakra-colors-purple-200` --> equivalent hex value change...